.travis.yml

```diff
@@ -17,13 +28,18 @@ script:
 - make test
 - make vet
 - make website-test
+- docker-compose -f $(pwd)/scripts/docker-compose.yml up -d
+- $(pwd)/scripts/wait-rabbitmq-docker.sh $(pwd)/scripts/docker-compose.yml
```

Comment:
You are passing the docker-compose file but it's not really used inside the script, I guess you can remove it?

Reply:
Indeed :+1: , a bit copy/pasted from another provider where it was used :sweat_smile: